doc/authentication.md

```diff
@@ -105,7 +105,7 @@ Alternatively, the clients can provide the token directly when adding the remote
 
 ### Using a PKI system
 
-In a {abbr}`PKI (Public key infrastructure)` setup, a system administrator manages a central PKI that issues client certificates for all the lxc clients and server certificates for all the LXD daemons.
+In a {abbr}`PKI (Public key infrastructure)` setup, a system administrator manages a central PKI that issues client certificates for all the LXD clients and server certificates for all the LXD daemons.
```

Comment:
This should be "LXD client and server", not "LXC client and server" - correct?

Reply:
Yes this makes sense. Just "clients" would probably have been fine too.